README.md

```diff
@@ -22,19 +22,19 @@ _Some similar features might have been implemented/merged_
 - Yarn/npm in favour of bower
   - Load only moment.js without languages (Languages are included in translations)
   - unneeded components removed (es6-shim, tablesort, numeraljs, leaflet-providers, jshashes)
-- RBush v2 - performance boost in last versions (Positions labels and clients on the map)
+- RBush v2 - performance boost in last versions (positions, labels and clients on the map)
 - Ruby dependency removed
 - FixedCenter is required
-- Sass-lint, scss and variables rewritten for easy customization/adjustments
+- Sass-lint, scss and variables rewritten for easy customizations/adjustments
 - Cross browser/device support improved (THX@BrowserStack)
 - Leaflet fork with a patch to avoid IE/Edge crashes
-- Yarn package manager in favor of npm (npm still works)
+- Yarn package manager in favour of npm (npm still works)
```

Comment:
We can switch from the American spelling to the British and general English one, but I didn't see any indication, that this was necessary, as we were using mostly American. But no objections.